src/controllers/auth.js

```diff
@@ -0,0 +1,24 @@
+const User = require('../models/User');
+
+const getUserById = async (req, res) => {
```

Comment:
[Naming / Standards / Namespacing]
This is a bit of a matter of opinion so decide what you like or we can decide as a group and just make it a standard.
Some would say not to use a name like `getUserById` inside the user controller because user is implicitly assumed from the user controller.

Another way of putting this is a caller of this controller might have code like this:
```javascript
const UserController = require('../controller/User');
...
router.use("user/<id>", UserController.getUserById);
...
```

So the `UserController.getUserById` is kinda redundant in that it says "user" twice.

If instead we implicitly namespace this function we can call it something like `getById` then the code in the above example is `UserController.getById`. 

In this example it seems a little pedantic to jump on this, but these can get unruly at times so you can end up with code like this: `UserController.getUserById(id).getUserPropertyByName(name).formatPropertyAsPascalCase().ToString()` compared to `UserController.getById(id).getPropertyByName(name).formatAsPascalCase().ToString()`.

Again, which is better is something of a matter of opinion so start forming your own 😄

Reply:
Replaced getUserById with UserController as recommended.

Reply:
I mean I replaced it in PRACT-16 as I will close this PR.